components/shared/CreateOrg/CompanyNameField.tsx

```diff
@@ -0,0 +1,26 @@
+import { FormField, FormItem, FormLabel, FormControl, FormMessage } from "@/components/ui/form";
+import { Input } from "@/components/ui/input";
+
+interface CompanyNameFieldProps {
+  control: any; // Replace with the appropriate type if available
```

Comment:
why any?

Reply:
i have removed unnecessary uses of any and commited it into the pull request